src/components/ManualWorkFlowDashboard.jsx

```diff
@@ -5,7 +5,8 @@ function ManualWorkFlowDashboard({backButtonString, alertText, searchTest, displ
     let navigate = useNavigate();
 
     return (
-       <div className='bg-bg h-screen' >
+       <div className='bg-grey-40 h-screen' >
```

Comment:
why two divs. Merge it into a single div.

Reply:
2 div's needed as we have to give bg colors for full screen and margin for only content.